### ISSUE TITLE
Seems "wasm-strip" is no more

### ISSUE THREAD

**Issue by mattleibow** [unoplatform/Uno.Wasm.Bootstrap#235]
This PR: https://github.com/mono/mono/pull/20006

I created a file to proxy this:

```bash
#!/bin/bash
DIRECTORY=$(cd `dirname $0` && pwd)

$DIRECTORY/wasm-opt --strip-dwarf $@ -o $@
```

**Comment by jeromelaban** [comment]
Thanks! At this point the mono build Uno is using is based on 1.39.11 explicitly, because of a Python3 issue https://github.com/emscripten-core/emscripten/issues/10551 (which may have been fixed in recent builds).

**Comment by jeromelaban** [comment]
Fixed in 1.4
